packages/tree-widget/src/components/TreeUiItemsProvider.tsx

```diff
@@ -0,0 +1,127 @@
+import { I18N } from "@bentley/imodeljs-i18n";
+import {
+  AbstractWidgetProps,
+  StagePanelLocation,
+  StagePanelSection,
+  UiItemsProvider,
+} from "@bentley/ui-abstract";
+import {
+  UiFramework,
+  SpatialContainmentTree,
+  ClassGroupingOption
+} from "@bentley/ui-framework";
+import React from "react";
+import { TreeWidgetComponent } from "./TreeWidgetComponent";
+import { CategoriesTreeComponent, ModelsTreeComponent, TreeWidgetControlOptions } from "../tree-widget-react";
+import { IModelConnection, Viewport } from "@bentley/imodeljs-frontend";
+import { SelectableContentDefinition } from "@bentley/ui-components";
+
+/** Provides the property grid widget to zone 9 */
+export class TreeUiItemsProvider implements UiItemsProvider {
```

Comment:
Maybe should be called `TreeWidgetUiItemsProvider`? Idk, probably too nit picky.
@calebmshafer thoughts?